model/nic.rb

```diff
@@ -14,7 +14,7 @@ class Nic < Sequel::Model
   include SemaphoreMethods
 
   semaphore :destroy, :start_rekey, :trigger_outbound_update,
-    :old_state_drop_trigger, :setup_nic, :repopulate, :lock
+    :old_state_drop_trigger, :setup_nic, :repopulate, :lock, :allocated
```

Comment:
Do we really need a new semaphore? Can't we just use `vm.allocated_at`?

Reply:
You're right, I updated it accordingly.